### ISSUE TITLE
md-input not reactive when initial value empty

### ISSUE THREAD

**Issue by deulos** [marcosmoura/vue-material#40]
The md-inputs do not acquire the value of the v-model if their initial value is empty. Example:

In template:
```
...
<input v-model="foo">

<md-input-container>
  <label>Label</label>
  <md-input type="text" v-model="foo" required></md-input>
</md-input-container>
...
```

In code:
```
...
data() {
  return {
    foo: null
  };
},

created() {
  setTimeout(() => {
    this.foo = 'bar'
  }, 2000);
}
...
```

The simple input will update it's value after 2 secs but the md-input will remain empty.

This does not happen when foo has any value with length (not null nor empty string).

**Comment by marcosmoura** [comment]
Oh. Quite strange. I'll take a deep look at this and I will come back with a fix. ASAP. 

Thank you for sending this.

**Comment by deulos** [comment]
No problem, I gues is due to not watching the value, so the class taking the label above is not applied

**Comment by damienix** [comment]
I have the same problem with combination with vuex. It looks like md-input is never updated when you update value programmatically using a model.

**Comment by marcosmoura** [comment]
Already implemented and will be shipped on the very next release (v0.3.0)
